sunpy/net/dataretriever/tests/test_kanzelhohe.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import datetime
+
+from astropy import units as u
+from sunpy.time.timerange import TimeRange
+from sunpy.net import attrs as a
+from sunpy.net.dataretriever.client import QueryResponse
+from sunpy.net.fido_factory import UnifiedResponse
+from sunpy.net import Fido
+import sunpy.net.dataretriever.sources.kanzelhohe as kanzelhohe
+
+KClient = kanzelhohe.KANZClient()
```

Comment:
This should be turned into a pytest fixture.

Reply:
Done